[2021-05-08 11:26:01,028] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-05-08T05:35:11.123810+00:00 [queued]>
[2021-05-08 11:26:01,037] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-05-08T05:35:11.123810+00:00 [queued]>
[2021-05-08 11:26:01,037] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 11:26:01,037] {taskinstance.py:1069} INFO - Starting attempt 2 of 2
[2021-05-08 11:26:01,037] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-08 11:26:01,170] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): also_run_this> on 2021-05-08T05:35:11.123810+00:00
[2021-05-08 11:26:01,173] {standard_task_runner.py:52} INFO - Started process 143283 to run task
[2021-05-08 11:26:01,179] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', '2021-05-08T05:35:11.123810+00:00', '--job-id', '41', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/etl/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpsbap8esw', '--error-file', '/tmp/tmpp8qundl9']
[2021-05-08 11:26:01,179] {standard_task_runner.py:77} INFO - Job 41: Subtask also_run_this
[2021-05-08 11:26:01,312] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.also_run_this 2021-05-08T05:35:11.123810+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-08 11:26:01,401] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T05:35:11.123810+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-05-08T05:35:11.123810+00:00
[2021-05-08 11:26:01,402] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-08 11:26:01,402] {bash.py:158} INFO - Running command: echo "run_id=manual__2021-05-08T05:35:11.123810+00:00 | dag_run=<DagRun example_bash_operator @ 2021-05-08 05:35:11.123810+00:00: manual__2021-05-08T05:35:11.123810+00:00, externally triggered: True>"
[2021-05-08 11:26:01,408] {bash.py:169} INFO - Output:
[2021-05-08 11:26:01,410] {bash.py:173} INFO - run_id=manual__2021-05-08T05:35:11.123810+00:00 | dag_run=<DagRun example_bash_operator @ 2021-05-08 05:35:11.123810+00:00: manual__2021-05-08T05:35:11.123810+00:00, externally triggered: True>
[2021-05-08 11:26:01,410] {bash.py:177} INFO - Command exited with return code 0
[2021-05-08 11:26:01,493] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20210508T053511, start_date=20210508T055601, end_date=20210508T055601
[2021-05-08 11:26:01,635] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-08 11:26:01,670] {local_task_job.py:146} INFO - Task exited with return code 0
